### PR TITLE
update docs

### LINKED ISSUE #23: Links to customizations header in README.md are missing #
There are two links in README.md that are intended to link to the the Customizations heading but are currently missing the hash mark so they try to link to a separate file and result in 404.

### PR DESCRIPTION
**Pull request by NormJohnIV**
closes #23

### CODE REVIEW

File: docs/CONFIG-VARS.md [hunk 1]
```diff
@@ -161,56 +161,56 @@ When setting V4_CFG_MANAGE_STORAGE to true, A new storage classes will be create
...
+| V4_CFG_POSTGRES_PORT | Existing postgres port | string | 5432 | false | | viya |
+| V4_CFG_POSTGRES_DATABASE | Existing postgres database name | string | "SharedServices" | false | | viya |
 
 ## LDAP / Consul
 
 | Name | Description | Type | Default | Required | Notes | Actions |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
-| V4_CFG_EMBEDDED_LDAP_ENABLE | Deploy openldap in the namespace for authentication | bool | false | false | [Openldap Config](../roles/vdm/templates/generators/openldap-bootstrap-config.yaml) | vdm |
-| V4_CFG_CONSUL_ENABLE_LOADBALANCER | Setup LB to access consul ui | bool | false | false | Consul ui port is 8500 | vdm |
+| V4_CFG_EMBEDDED_LDAP_ENABLE | Deploy openldap in the namespace for authentication | bool | false | false | [Openldap Config](../roles/viya/templates/generators/openldap-bootstrap-config.yaml) | viya |
```
Comment (thpang) [inline docs/CONFIG-VARS.md:198]: You also changed the directory here was the intentional?

Comment (NormJohnIV) [reply docs/CONFIG-VARS.md:198]: No, that was not intentional

Comment (thpang) [reply docs/CONFIG-VARS.md:198]: I'll wait for the update, then re-check.

Comment (NormJohnIV) [reply docs/CONFIG-VARS.md]: updated

**Pull request merged**